ops/terraform/services/base/values/ephemeral.yaml

```diff
@@ -1,3 +1,4 @@
+# CIPHER FILE HEADER, DO NOT REMOVE: <<CIPHER>>AgV4wCYoE580UxS5VbF6Xnqo8GjTwdnJiZ0I0l1RhhzSg5YAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREE4cnAvS2RZRU13eGY2VUt3ZkxCZkVpM1ZEZno4bGNkemc5dzRzakhNNXZoWEZMd0FtcGpNMHVmWnc5L0VubEVaZz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay0xNWZhNGU2MzcxOTI0YmZmYTk3NjRjMjE4ZjdhNzY3YQC4AQIBAHic22-xSw25t2N5FjIsl0BfswHAJC80QDtF4qY0HagMgAEA46egFuOJH-_7D5AY5T5fAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMMDLdOHXXYdj5_eyNAgEQgDtaGv5Z3awF4O1cLfpiCukEdk64o2wfghQh6JNgm49842OaU3nnMMvExlWmntVYvxfsDmClbdqqcy9JjgIAABAAI-0H2NDJCNypG7qy4zRj1bR8t6YDgm1k-YHVDjXMF4pYocOmrNsAhWgl_xwOLltE_____wAAAAEAAAAAAAAAAAAAAAEAAAAEQ2mrRYTK6v42FOSiiTDXHEBNOAYAZzBlAjEAzVV0FZTXGoe13LN3_OTFfm_jASxlJL3cPj5dA5MXKgXA_diICP47qawxNRSkCjCRAjBy5RwC-wW3Ru1AwX_He_EU_OAsLQwvkXViNBelq3gxK5hxRMUYxlu7zoUYihjbwVE=<</CIPHER>>
```

Comment:
I wasn't sure if the cipher text included in the headers had any significance or not. I don't see any code that actually uses it so I'm thinking it does not.

Reply:
Along with the updates to the `.sensitive-files` you've submitted here, there's some logic that looks for this in the pre-commit hook (Historically, installation required users to run a separate `mvn init`, but I _think_ that's installed automagically with most any maven invocation these days. See for yourself under working copy-relative `.git/hooks/pre-commit`). It's there in an attempt to save us from committing whole-sale decrypted files...

Reply:
Yes, it looks for `CIPHER FILE HEADER, DO NOT REMOVE` but the context of the <<CIPHER>> bit on the comment header seems to be irrelevant.

Reply:
It's not a perfect safety net, but when _decrypted_ that CIPHER block should yield "<<SECURE>> CIPHER ENCRYPTED FILE HEADER; SHOULD BE ENCRYPTED<</SECURE>>" which should signal to the pre-commit hook that it's a decrypted file and fail the commit.

How do you edit the `ephemeral.yaml` with cipher?

Reply:
Oh I see. I'll fix that. I used the `cipher.main.kts` script to encrypt the file in the first place. I can add some notes to the README about this too since I didn't see much information about this.